blueprints/policies/auto-scaling/base/config-defaults.libsonnet

```diff
@@ -0,0 +1,78 @@
+local auto_scaling_base_defaults = {
+  policy_name: '__REQUIRED_FIELD__',
+
+  components: [],
+
+  resources: {
+    flow_control: {
+      classifiers: [],
+    },
+  },
+
+  evaluation_interval: '1s',
+};
+
+local promql_scale_out_controller_defaults = {
+  query_string: '__REQUIRED_FIELD__',
+  threshold: 1.0,
+  gradient: {
+    slope: -1.0,
+  },
+};
+
+local promql_scale_in_controller_defaults = {
+  query_string: '__REQUIRED_FIELD__',
+  threshold: 0.5,
+  gradient: {
+    slope: 1.0,
```

Comment:
Slope defines the direction of correction i.e. whether the scale should increase/decrease when signal is greater than set point. Default slope should be the same for both scale in and scale out controllers. The default slope is 1 for both in the proto spec. This works well for utilization metrics, e.g. if cpu is greater than threshold then scale out

Reply:
Got it. Updated the value to 1.0. Thanks!